lib/fastlane/plugin/bundletool/actions/bundletool_action.rb

```diff
@@ -158,30 +158,37 @@ def self.puts_error!(message)
       def self.available_options
         [
           FastlaneCore::ConfigItem.new(key: :ks_path,
+                                       env_name: 'FL_BUNDLETOOL_KEYSTORE_FILE',
                                        description: 'Path to .jks file',
                                        is_string: true,
                                        optional: true),
           FastlaneCore::ConfigItem.new(key: :ks_password,
+                                       env_name: 'FL_BUNDLETOOL_KEYSTORE_PASSWORD',
```

Comment:
Same, changed from `FL_BUNDLETOOL_STORE_PASSWORD` to `FL_BUNDLETOOL_KEYSTORE_PASSWORD`